quickstarts/azure/azure-event-grid-subscriptions/config.yml

```diff
@@ -0,0 +1,43 @@
+slug: azure-event-grid-subscriptions
+description: |-
+  ## What is Azure Event Grid Subscriptions?
+
+  Azure Event Grid Subscriptions tells Event Grid which events on a topic you're interested in receiving. The endpoint or built-in mechanism to route events, sometimes to more than one handler. Subscriptions are also used by handlers to intelligently filter incoming events.
+
+  ### New Relic Azure Event Grid Subscriptions quickstart features
+
+  A standard dashboard that tracks key indicators like delivery failed events, delivered events, dropped events and matched events. It runs custom queries and visualizes the data immediately
+
+
+  ### Why monitor Azure Event Grid Subscriptions with New Relic?
+
+  [New Relic Azure Event Grid Subscriptions](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-event-grid-subscription-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure event grid subscriptions via different metrics including delivery failed events, delivered events, dropped events and matched events.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Event Grid Subscriptions is critical to track the performance through key metrics. Download New Relic Azure Event Grid Subscriptions monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Event Grid Subscriptions Service.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+  - New Relic Partner
+title: Azure Event Grid Subscriptions
+documentation:
+  - name: Azure Event Grid Subscriptions installation docs
+    description: |
+      Monitor Azure Event Grid Subscriptions by connecting Azure to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-event-grid-subscription-monitoring-integration/
+keywords:
+  - azure
+  - event grid
+  - event grid subscriptions
+  - product partnerships
+  installPlans:
```

Comment:
This is indented incorrectly. Please remove it since we have deprecated `installPlans` anyways.